spec/spec.ddoc

```diff
@@ -3,6 +3,36 @@ ROOT = ..
 SEARCHDEFAULT_SPEC = selected
 DDOC_BLANKLINE = $(DIVC blankline)
 
+BODY_PREFIX =
```

Comment:
Ditto, this is not the correct place for this either.

1. In spec.ddoc (this file), add `BODYCLASS=doc spec`
2. In style.css, add these rules, but prefixed with `body.spec` (i.e. `body.spec p:before { ... }`)